psi-probe-core/src/main/java/psiprobe/controllers/AbstractTomcatContainerController.java

```diff
@@ -70,4 +80,24 @@ public String getViewName() {
   public void setViewName(String viewName) {
     this.viewName = viewName;
   }
+
+  /**
```

Comment:
This code is a combination of two methods from tomcat.  They are both removed.  Applied them together in this case class so it behaves the same as it did.  I've migrated to new FileUpload usage as tomcat did as well but needed this still.  Maybe there is a better way but this was quick and dirty solution.